standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Text.Json.xml

```diff
@@ -543,7 +543,7 @@ The current <paramref name="reader" /> token does not start or represent a value
       <param name="utf8Text">The UTF-8 encoded text to compare against.</param>
       <returns>
         <see langword="true" /> if the string value of this element has the same UTF-8 encoding as
-    <paramref name="utf8Text" />; otherwise, <see langword="false" />.</returns>
+<paramref name="utf8Text" />; otherwise, <see langword="false" />.</returns>
```

Comment:
This spacing change is weird. Where did it come from.

Reply:
Downloaded from Docs. I don't touch the files.

Reply:
The unexpected endline may have been ported like that with the docs porting tool some time ago, because some programmers tend to write their triple slash comments with lines that are no longer than 80 characters. It used to be a restriction in older languages, but not on C#, so I guess people still do it to reduce the need to scroll horizontally.

The docs porting tool used to port these messages untouched, but I fixed that bug some time ago: any unexpected endline that is not preceded by a symbol (.,?!*:) is removed automatically before porting.

So maybe this unexpected endline is a remnant of how the docs porting tool used to port strings before I fixed that.

I wouldn't worry. I noticed there are a lot of these, and some of these precede the docs porting tool.